homeassistant/components/ihc/sensor.py

```diff
@@ -64,12 +71,12 @@ def device_class(self):
         )
 
     @property
-    def native_value(self):
+    def native_value(self) -> StateType | date | datetime | Decimal:
         """Return the state of the sensor."""
         return self._state
 
     @property
-    def native_unit_of_measurement(self):
+    def native_unit_of_measurement(self) -> str | None:
```

Comment:
This is type hint improvement - unrelated to config-flow. Please move to preliminary/follow-up PR.

Reply:
Done in PR [90350](https://github.com/home-assistant/core/pull/90350)